config/routes.rb

```diff
@@ -64,6 +64,12 @@
         end
       end
     end
+
+    namespace :v3 do
+      resources :builds, only: [], param: :timestamp do
+        resource :split_registry, only: :show, controller: 'split_registries'
```

Comment:
i think specifying the controller is unnecessary. it should be able to infer that from the resource name